src/migrations/changeGameTypePKtoInt.sql

```diff
@@ -0,0 +1,60 @@
+ALTER TABLE public."Game" DROP CONSTRAINT "Game__gameTypeId_fkey";
+ALTER TABLE public."GameItemAvailability" DROP CONSTRAINT "GameItemAvailability__gameTypeId_fkey";
+ALTER TABLE public."Achievement" DROP CONSTRAINT "Achievement__gameTypeId_fkey";
```

Comment:
in order to make the migration, had to make some chagnes throughtout some constraints in other tables